### PR TITLE
Update Moveable to fix right-click handling + resizing

### LINKED ISSUE #1011: Image resizing not working when the origin point is in the top-left corner
## Bug Description

When the origin of the resizing is top-left corner AND the image is rotated 90 degrees, resizing fails and the image "disappears".

Similar issues occur with text boxes and many different rotations (when rotated more than 90 degrees from 0 in either direction)

## Expected Behaviour
Resizing should work despite of the rotation / start position

## Steps to Reproduce

1. Add an image
2. Assign rotation 90 degrees
3. Position it to the top-left corner
4. Resize from the bottom-right corner as it is visible currently without considering rotation
5. See the odd stuff happening.

## Screenshots
image-lost

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## QA Instructions
1. Add an image
2. Set rotation to 90 degrees
3. Now drag the image to the upper left corner
4. Resize it from the corners
5. Verify the resizing works as expected and that the image doesn't "jump around".

### LINKED ISSUE #2087: Right-click on a shape causes a drag to start and continue while the mouse is not down.
## Bug Description

This is pretty weird, found during APAC bugbash: Right-click on a shape causes a drag to start and continue while the mouse is not down.

## Expected Behaviour

Expected the right click to do nothing other than bring up the generic browser right-click menu.

## Steps to Reproduce

Place a shape on the page
Right click on the shape and release the right click
Move mouse around. 

## Additional Context

- Plugin Version:
- Operating System:
- Browser:

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

### PR DESCRIPTION
**Pull request by miina**
## Summary

Prevents dragging when the user is either using right-click or using Ctrl when clicking.

## Relevant Technical Choices
Right-click without using an actual mouse is different from using it on a laptop without a mouse.
This PR prevents dragging when Ctrl is down.

## To-do

-

## User-facing changes

When using right-click or Ctrl + Left-click for opening the context menu, the drag event is not started.

## Testing Instructions
I
1. Add a shape
2. Open the context menu on the shape by using Ctrl + left click
3. Close the context menu by clicking somewhere outside
4. Move the mouse around
5. Verify that dragging is not happening.

II
1. Add an image
2. Set rotation to 90 degrees
3. Now drag the image to the upper left corner
4. Resize it from the corners
5. Verify the resizing works as expected and that the image doesn't "jump around".

---

Fixes #2087 
Fixes #1011

### CODE REVIEW

File: assets/src/edit-story/components/canvas/multiSelectionMovable.js [hunk 1]
```diff
@@ -231,11 +231,16 @@ function MultiSelectionMovable({ selectedElements }) {
         });
       }}
       onDragGroupStart={({ events, inputEvent }) => {
+        // Prevent dragging on right-click.
+        if (inputEvent.ctrlKey || 2 === inputEvent.button) {
```
Comment (miina) [inline assets/src/edit-story/components/canvas/multiSelectionMovable.js]: @barklund @dvoytenko 
Is there any reason to not prevent dragging while Ctrl is held down?

Looks like the issue of dragging being started unexpectedly happens currently due to opening the context menu via using Ctrl + left click. This logic prevents it from happening, however, wondering if we should restrict it strictly for some reason, e.g. `if (inputEvent.ctrlKey && inputEvent.button === 0) return false;` instead.

Thoughts?

Comment (miina) [reply assets/src/edit-story/components/canvas/multiSelectionMovable.js]: FI: There's also [a bug filed under Moveable repo](https://github.com/daybrush/moveable/issues/252) so hopefully we can remove this logic from our code at some point.

Comment (miina) [reply assets/src/edit-story/components/canvas/multiSelectionMovable.js]: Looks like it got fixed there already!

**Review by swissspidy — APPROVED** [review]

**Comment by obetomuniz** [comment]
Hey @miina, will this update fix #2101 as well? If yes, it is just perfect!

**Comment by miina** [comment]
@obetomuniz Yes, #2101 seems to be the exact duplicate of #1011, so will close it as such.

**Comment by swissspidy** [comment]
Can you update Moveable to 0.20.10?

**Review by merapi — APPROVED** [review]

**Pull request merged**